src/templates/slack-page.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect } from 'react'
+import PropTypes from 'prop-types'
+import { graphql } from 'gatsby'
+import Layout from '../components/Layout'
+import Section from '../components/Section/Section'
+import SectionTitle from '../components/SectionTitle/SectionTitle'
+import SectionDescription from '../components/SectionDescription/SectionDescription'
+import CustomLink from '../components/CustomLink/CustomLink'
+
+export const SlackPageTemplate = ({
+  title,
+  description,
+  slackURL,
+  linkText,
+}) => {
+  useEffect(() => {
```

Comment:
when will this effect happen or does user have even chance to read the redirect message.

Reply:
It will run on mount. Redirect message is only if this fails. It's actually preferrable to not see any message at all.

Reply:
when can it fail if it is run on mount and part of react.

Reply:
It can fail if browser is configured not to follow redirects